### DISCUSSION TITLE
Using v.voronoi.skeleton in Grass8 with QGIS?

### DISCUSSION THREAD

**Discussion by stefankinell** [OSGeo/grass#3422] [Q&A] [answered]
I hope this question lands in the right place since it is a Grass and QGIS-question.

In previous QGIS-versions that used Grass7 I designed a model that used v.voronoi.skeleton. But with the upgrade to later versions, e.g. 3.43, and Grass8, this model will not work anymore. It complaints that grass7:v.voronoi.skeleton is no longer available. Which is not strange since now Grass8 is installed. When trying to find the equivalent module in grass8 I fail to do so. I find references to running v.voronoi -s in command line, but no function i qgis.

Is there a simple way of solving this? I have tried to look at other skeleton-plugins but can not find anyone that works as a replacement to the v.voronoi.skeleton. Is it possible to install an older grass7 and use that with a newer QGIS?

**Comment by petrasovaa** [comment, marked as answer]
GRASS v.voronoi has not changed API between 7 and 8. There has never been v.voronoi.skeleton in GRASS, it is a QGIS wrapper. You can use v.voronoi directly in GRASS with `-s` flag to get the skeletons. 

So I would ask QGIS developers. You can let them know they can follow up here if there is anything needed on the GRASS side.

**Comment by stefankinell** [comment]
Thank you for that clarification @petrasovaa . Then I should see if I can find some persons on the qgis-side to adress the question to, or learn how to rewrite my qgis-model into a script so I can utalize GRASS directly and more efficient.

**Comment by neteler** [comment]
See also
https://github.com/qgis/QGIS/issues/56365
